### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by nicholebarry**
Update documentation since it's been awhile.

### CODE REVIEW

File: outputs.md
Comment (mwilensky768) [inline outputs.md:222]: Not sure why an axis is being referred to by a number. Can these four (`obsx`, `obsy`, `zenx`, `zeny`) be clarified a bit? If unclear for now, maybe find a way of marking for clarification later since I'd prefer this didn't hold up the merge.

File: outputs.md
Comment (mwilensky768) [inline outputs.md:311]: Since there are multiple sensible ways to make complex things real, can there be a comment about how that constraint is done (take real part of amplitude or something more sophisticated)?

File: outputs.md
Comment (mwilensky768) [inline outputs.md:356]: See comment for fits files on line 311.

File: outputs.md
Comment (mwilensky768) [inline outputs.md:149]: I had quite a bit of confusion when trying to outline these below. I've marked them for help. If these are obvious to someone, maybe fill them in now. Otherwise leave as is.

**Review by mwilensky768 — CHANGES_REQUESTED** [review]
I left a few comments that I feel should be clarified now. Comments about `fhd_struct_init` can probably just be responded to without code changes. For documentation changes, if clarification is requested, I mean for it to be provided in the documentation now if it is very easy to figure out, otherwise I think it would be good to mark with `!Q` before merging.

File: dictionary.md
Comment (nicholebarry) [reply dictionary.md]: nice catch

File: fhd_core/setup_metadata/fhd_struct_init_obs.pro
Comment (nicholebarry) [reply fhd_core/setup_metadata/fhd_struct_init_obs.pro]: Unused keyword

File: outputs.md
Comment (nicholebarry) [reply outputs.md]: These are extents, i.e. the max value in a certain direction. Modified. Hope that clears it up.

**Comment by nicholebarry** [comment]
@mwilensky768 addressed your comments, see if that is satisfactory for you 

Also saw that no_png wasn't actually passed to the cal procedure, and I've added a keyword I've been using a lot lately (save_beam_metadata_only). Useful for when you don't want to write 10Gb worth of beam pointers to disk.

**Review by mwilensky768 — COMMENTED** [review]
Thanks for addressing my comments (and cleaning up some of my mess!). It looks like maybe there was a rebase? Was this to grab the `save_beam_metadata_only` functionality from a different branch? If so, can you just check that one more time by eye and let me know if the rebase went OK? It looks sane to me. Happy to approve if you're confident that didn't break anything.

**Comment by nicholebarry** [comment]
I rebased since documentation sometimes gets put in master. Nothing more than just trying to get the full picture. 

Nothing broke.

**Review by mwilensky768 — APPROVED** [review]

**Pull request merged**
